graphstorm-processing/graphstorm_processing/config/config_parser.py

```diff
@@ -21,7 +21,11 @@
 from graphstorm_processing.constants import SUPPORTED_FILE_TYPES
 from .label_config_base import LabelConfig, EdgeLabelConfig, NodeLabelConfig
 from .feature_config_base import FeatureConfig, NoopFeatureConfig
-from .numerical_configs import MultiNumericalFeatureConfig, NumericalFeatureConfig
+from .numerical_configs import (
+    MultiNumericalFeatureConfig,
+    NumericalFeatureConfig,
+    BucketFeatureConfig,
```

Comment:
It's a good idea to try to keep multiple imports alphabetically sorted, see https://google.github.io/styleguide/pyguide.html#s3.13-imports-formatting

```suggestion
    BucketFeatureConfig,
    MultiNumericalFeatureConfig,
    NumericalFeatureConfig,
```